package-lock.json

```diff
@@ -0,0 +1,8989 @@
+{
```

Comment:
Since we're using `yarn.lock` for this project, I think we should remove this and it to `.gitignore` for now.